backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/BPBankAcctUse.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.bpartner.service;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import lombok.NonNull;
+import org.compiere.model.X_C_BP_BankAccount;
+
+import java.util.Arrays;
+
+@JsonAutoDetect(fieldVisibility = Visibility.ANY, getterVisibility = Visibility.NONE, isGetterVisibility = Visibility.NONE, setterVisibility = Visibility.NONE)
+public enum BPBankAcctUse
+{
+	NONE(X_C_BP_BankAccount.BPBANKACCTUSE_Nichts),
+	BOTH(X_C_BP_BankAccount.BPBANKACCTUSE_Both),
```

Comment:
instead of BOTH i think it would make more sense to call the enum as DEBIT_OR_DEPOSIT.
@metas-ts  wdyt?

Reply:
@teo @adi-stefan don't aggree but to me i doesn't see critical because if you lool at "both", you can easily see that "debit" and "deposit" are two of the other option..
also, it's B in the DB and changing that would be even more effort and FUD--

Reply:
@metas-ts  there is also PROVISION (the 3rd one)

Reply:
Changed enum name to `DEBIT_OR_DEPOSIT`. No change to the existing DB value.